metadata-ws/src/test/resources/features/security.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Security related tests
+
+  Scenario: access security disabled links
+    When I request unauthorized GET /
```

Comment:
instead of an unauthorized user, it should be an anonymous user

Reply:
sure